third_party/googleapis/google/api/README.md

```diff
@@ -0,0 +1,5 @@
+This folder contains the schema of the configuration model for the API services
```

Comment:
@jskeet I'm not sure if we have some process to follow to import third party protos. If so, let us know.

Reply:
I would hope we wouldn't need anything from `google/api`. The only import is for field_behavior.proto, which we shouldn't use. I'm hoping this PR can turn into three files - data.proto and events.proto as new files, and an update to the README.

Reply:
@jskeet If we don't import `field_behavior.proto`, the build will fail as follows: https://travis-ci.org/github/googleapis/google-cloudevents/builds/714923575
Is that expected?

Reply:
That failure shows you still trying to import the proto. You should:

- Remove the proto
- Remove the import
- Remove all uses of the annotation

Basically it's an annotation that only makes sense in an API proto, which this isn't.